server/index.ts

```diff
@@ -1,21 +1,36 @@
+import dotenv from 'dotenv';
+dotenv.config();
 import express from 'express';
-import path from 'path';
+import { connectDatabase, getEntryCollection } from './database';
+
+if (!process.env.MONGODB_URI) {
+  throw new Error('No MongoDB URI dotenv variable');
+}
 
 const port = process.env.PORT || 3001;
 const app = express();
+app.use(express.json());
 
 app.get('/api/hello', (_request, response) => {
   response.json({ message: 'Hello from server' });
 });
 
-// Serve production bundle
-app.use(express.static('dist'));
```

Comment:
This was required to serve the client bundle.

Reply:
done 👍